.github/scripts/excluded_from_copyright

```diff
@@ -48,5 +48,6 @@
 ./pkg/assembler/graphql/resolvers/certifyVEXStatement.resolvers.go
 ./pkg/assembler/graphql/resolvers/path.resolvers.go
 ./pkg/assembler/graphql/generated/path.generated.go
+./pkg/assembler/graphql/generated/metadata.generated.go
```

Comment:
Adding a comment to block the PR.. I just realized that timestamp probably will be important here, thoughts @pxp928 @mihaimaruseac

Reply:
+1

Reply:
Yeah, let's record timestamps

Reply:
done!